admin/src/Environment.js

```diff
@@ -49,29 +55,33 @@ class Environment extends Component {
 
     render() {
         const {
-            state: { config, edition, error, loading, newConfig },
+            state: { edition, error, loading, newConfig, filteredConfig },
             effects: { setNewConfig, toggleEdition },
+            setSearch,
         } = this.props;
 
         return (
             <div style={styles.container}>
                 <Card>
-                    <CardText>
-                        {!edition && <RaisedButton label="Edit" primary onClick={toggleEdition} />}
-                        {edition && <RaisedButton label="Save" primary onClick={this.handleSaveClick} />}
-                        {edition && <RaisedButton label="Cancel" onClick={toggleEdition} />}
-                    </CardText>
+                    <CardActions style={styles.actions}>
+                        <div>
```

Comment:
I think it would be better to move the edit button to the end and the search at the beginning because search is more important and will probably be used more often. Moreover, this full edition mode should be discouraged as potentially dangerous and we will introduce item edition soon (see #83)

But we should leave the Save and Cancel at the beginning in edition mode.